getSimulationPA.m

```diff
@@ -49,15 +50,14 @@
         if(nderiv>0)
             switch Model.exp{s}.scale
                 case 'log'
-                    nt = size(SP.dmydxi,1);
-                    np = size(SP.dmydxi,4);
-                    ny = size(SP.dmydxi,2);
-                    % To be checked!
+                    nt = size(SP.dCydxi,1);
+                    np = size(SP.dCydxi,4);
+                    ny = size(SP.dCydxi,2);
```

Comment:
Here again the same... I don't think Cy is really needed...
Maybe I'm wrong, but this needs a math argument, if we really need it...